Terasology/WildAnimals
# feat: Add new animals - GCI 2019

### Contains
- New `cow` and `cat` animals created as part of GCI 2019 by @ujjman 
- New prefabs for wolf and deer using the `territorialCritter` behavior created by @BenjaminAmos

### Objective
This pull request implements two new creatures: `cat` and `cow`. It also includes two new prefabs showcasing the `territorialCritter` behavior ([Behaviors - PR#29](https://github.com/Terasology/Behaviors/pull/29)).

#### Testing Instructions:
- For the new creatures: use the console commands `spawnPrefab cow` and `spawnPrefab cat` in different locations to spawn the initial creatures.
- For the territorial prefabs: 
	- Use the console commands `spawnPrefab territorialDeer` and `spawnPrefab territorialWolf` in different locations to spawn the initial creatures
	- A player should approach one of the spawned creatures
	- The creature should attempt to attack the player but will stop as soon as the player reaches a certain distance from the creature's spawn point
	- Execute the command `assignGroupBehavior`
	- The spawned creatures should still attack the player when approached
